feature/visited/src/main/kotlin/uk/govuk/app/visited/ui/EditVisitedScreen.kt

```diff
@@ -0,0 +1,341 @@
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.input.nestedscroll.nestedScroll
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import uk.govuk.app.design.ui.component.BodyRegularLabel
+import uk.govuk.app.design.ui.component.CardListItem
+import uk.govuk.app.design.ui.component.LargeVerticalSpacer
+import uk.govuk.app.design.ui.component.ListHeadingLabel
+import uk.govuk.app.design.ui.component.MediumHorizontalSpacer
+import uk.govuk.app.design.ui.component.SmallVerticalSpacer
+import uk.govuk.app.design.ui.component.SubheadlineRegularLabel
+import uk.govuk.app.design.ui.component.Title2BoldLabel
+import uk.govuk.app.design.ui.theme.GovUkTheme
+import uk.govuk.app.visited.R
+import uk.govuk.app.visited.VisitedUiState
+import uk.govuk.app.visited.VisitedViewModel
+
+@Composable
+internal fun EditVisitedRoute(
+    onBack: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    val viewModel: VisitedViewModel = hiltViewModel()
+    val uiState by viewModel.uiState.collectAsState()
+
+    EditVisitedScreen(
+        uiState = uiState,
+        onEditPageView = { viewModel.onEditPageView() },
+        onBack = onBack,
+        modifier = modifier
+    )
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun EditVisitedScreen(
+    uiState: VisitedUiState?,
+    onEditPageView: () -> Unit,
+    onBack: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    LaunchedEffect(Unit) {
+        onEditPageView()
+    }
+
+    val scrollBehavior = TopAppBarDefaults.pinnedScrollBehavior(rememberTopAppBarState())
+
+    val visitedItems = uiState?.visited
+    val titleText = stringResource(R.string.visited_items_title)
+    val doneText = stringResource(R.string.visited_items_done_button)
+    val selectText = stringResource(R.string.visited_items_select_all_button)
+    val removeText = stringResource(R.string.visited_items_remove_button)
+
+    Scaffold(
+        containerColor = GovUkTheme.colourScheme.surfaces.background,
+        modifier = modifier
+            .nestedScroll(scrollBehavior.nestedScrollConnection)
+            .fillMaxWidth(),
+
+        topBar = {
+            TopNavBar(
+                title = titleText,
+                doneText = doneText,
+                onBack = onBack,
+                scrollBehavior = scrollBehavior,
+                modifier = modifier
+            )
+        },
+        bottomBar = {
+            BottomNavBar(
+                selectText = selectText,
+                removeText = removeText,
+                modifier = modifier
+            )
+        }
+    ) { innerPadding ->
+        LazyColumn (
```

Comment:
Noticed a couple of places need formatting in this file so suggest a 'Option + Command + L' to correct

Reply:
Fixed up